scalableminds/webknossos
# Add organization to user drop down menu

This PR adds an item with the user's organization to the user drop down menu.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- login as a user
- open the user drop down menu in the upper right corner
- your organization should be displayed 

### Issues:
- fixes #4169

------
